sdk/include/psdk/dsound.h

```diff
@@ -33,82 +33,104 @@ extern "C" {
 
 #ifndef DX_SHARED_DEFINES
 
-typedef float D3DVALUE, *LPD3DVALUE;
+    typedef float D3DVALUE, *LPD3DVALUE;
```

Comment:
Wine also doesn't have this indent. Did this file get caught in some auto-formatter?